Ensconce/Program.cs

```diff
@@ -182,147 +181,151 @@ private static void SetUpAndParseOptions(string[] args)
             var showHelp = false;
             var p = new OptionSet
                         {
-                            {"i|stdin", "Read template string from StdIn", s => readFromStdIn = s != null},
-                            {"h|help", "Show this message and exit", s => showHelp = s != null},
+                            {
+                                "i|stdin",
+                                "Read template string from StdIn",
+                                s => readFromStdIn = s != null
+                            },
+                            {
+                                "h|help",
+                                "Show this message and exit",
+                                s => showHelp = s != null
+                            },
                             {
                                 "w|webservice=",
```

Comment:
@richard-green as you're working in this area we could perhaps lose this parameter as well